downstream/attributes/attributes.adoc

```diff
@@ -101,3 +101,7 @@
 :OCPLatest: 4.10
 :ODF: Red Hat OpenShift Data Foundation
 :ODFShort: OpenShift Data Foundation
+
+// Red Hat products
+:RHSSO: Red Hat Single Sign-On
+:RHSSOOperator: Red Hat Single Sign-On Operator
```

Comment:
```suggestion
:OperatorRHSSO: Red Hat Single Sign-On Operator
```

Reply:
I think this should follow the convention of the other [Operator attributes](https://github.com/RedHatInsights/red-hat-ansible-automation-platform-documentation/blob/343e0a70fc5489d6f70faf5f04531985b2961346/downstream/attributes/attributes.adoc#L22)

Reply:
@oraNod Done. Thanks.